src/main/webapp/assets/js/controllers/controllers.js

```diff
@@ -232,12 +232,15 @@ canvasSupportApp.controller('addUserController', ['Friend', '$scope', '$rootScop
           $scope.userAvailable = true;
           $scope.done = true;
         } else {
+          $scope.loading2 = false;
```

Comment:
Suggestion: instead of loading and loading2 could these be more descriptive variables. For instance friendLookupLoading and friendCreateLoading.